src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const ApiError = require('../exeptions/api.error.js');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
+
+  res.statusCode = 500;
+
+  res.send({
+    message: 'Server error',
+  });
```

Comment:
The server error response is sent unconditionally, which means it will always be executed after handling an `ApiError`. Consider using an `else` statement to ensure the server error response is only sent if the error is not an instance of `ApiError`.